### ISSUE TITLE
Small icons with Kitty+Fira Code

### ISSUE THREAD

**Issue by refaelsh** [kyazdani42/nvim-tree.lua#1486]
### Description

I am seeing very small icons. Here is a screenshot:

![2022-08-03_23-38](https://user-images.githubusercontent.com/2750775/182707263-43dd527b-7fb7-459f-9408-2208ec561156.png)

Please help :-)

### Neovim version

```text
NVIM v0.7.2
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by nixbld

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "
/nix/store/lw4rkrcdhwnijxhix7fxfg3lfdpsa6jm-neovim-unwrapped-0.7.2/share/nvim
"

Run :checkhealth for more info
```

### Operating system and version

NixOS

### nvim-tree version

latest

### Minimal config

```lua
vim.cmd [[set runtimepath=$VIMRUNTIME]]
vim.cmd [[set packpath=/tmp/nvt-min/site]]
local package_root = "/tmp/nvt-min/site/pack"
local install_path = package_root .. "/packer/start/packer.nvim"
local function load_plugins()
  require("packer").startup {
    {
      "wbthomason/packer.nvim",
      "kyazdani42/nvim-tree.lua",
      "kyazdani42/nvim-web-devicons",
      -- ADD PLUGINS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
    },
    config = {
      package_root = package_root,
      compile_path = install_path .. "/plugin/packer_compiled.lua",
      display = { non_interactive = true },
    },
  }
end
if vim.fn.isdirectory(install_path) == 0 then
  print "Installing nvim-tree and dependencies."
  vim.fn.system { "git", "clone", "--depth=1", "https://github.com/wbthomason/packer.nvim", install_path }
end
load_plugins()
require("packer").sync()
vim.cmd [[autocmd User PackerComplete ++once echo "Ready!" | lua setup()]]
vim.opt.termguicolors = true
vim.opt.cursorline = true

-- MODIFY NVIM-TREE SETTINGS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
_G.setup = function()
  require("nvim-tree").setup {}
end
```

### Steps to reproduce

Just open Neovim with nvim-tree.

**Comment by alex-courtis** [comment]
Not an nvim-tree issue. See [nvim-web-devicons](https://github.com/kyazdani42/nvim-web-devicons).

However, it is not an nvim-web-devicons issue either.

The issue is likely with your font, which is displaying the icons at the intended size. Try another [patched font](https://www.nerdfonts.com/).

I use [hack](https://sourcefoundry.org/hack/); you can see it in action [here](https://github.com/kyazdani42/nvim-tree.lua#a-file-explorer-for-neovim-written-in-lua).

**Comment by refaelsh** [comment]
Thank you very much.
The `nvim-web-devicons` solved it. Here: https://github.com/kyazdani42/nvim-web-devicons/issues/148.

TL;DR version: change terminal font from `Fira Code` to `FiraCode Nerd Font`.
